Makefile

```diff
@@ -137,7 +137,7 @@ db-extras-zynq7:
 	+source minitests/roi_harness/zybo-swbut.sh && $(MAKE) -C fuzzers part_only
 	+source minitests/roi_harness/zybo-swbut.sh && \
 		$(MAKE) -C minitests/roi_harness \
-			HARNESS_DIR=database/artix7/harness/zybo/swbut run
```

Comment:
Thus should probably be `run copy`?

Reply:
Well, I was about to ask about that.
The minitests/roi_harness/Makefile is:
<pre>
copy: $(HARNESS_FILES)
    mkdir -p $(HARNESS_DIR)
    cp $(HARNESS_FILES) $(HARNESS_DIR)

run:
    make clean
    make copy
</pre>
So why do we additionally explicitly call ``run copy`` in artix7's db-extras?

Reply:
Dunno...

Reply:
@litghost I think you added the ``run copy`` to db-extras-artix7`` (correct me if I am wrong). Maybe we should remove the redundant ``copy`` there and this one is good to merge.